examples/06-verif-manual/vm-002-beam_stresses_and_deflections.py

```diff
@@ -0,0 +1,188 @@
+mapdl.verify()
+mapdl.prep7()
+
+###############################################################################
+# Define Element Type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element type (a beam-type).
+
+mapdl.antype("STATIC")
+mapdl.et(1, "BEAM188")
+mapdl.keyopt(1, 9, 3)  # Output at 9 intermediate locations
+mapdl.keyopt(1, 3, 3)  # Cubic shape function
+
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material.
+
+mapdl.mp("EX", 1, 30E6)
+mapdl.mp("PRXY", 1, 0.3)
+
+###############################################################################
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section properties for a beam element.
+
+w_f = 1.048394965
+w_w = 0.6856481
+sec_num = 1
+mapdl.sectype(sec_num, "BEAM", "I", "ISection")
+mapdl.secdata(15, 15, 28 + (2 * w_f), w_f, w_f, w_w)
+
+###############################################################################
+# Define Geometry:
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set up the nodes and elements. Create nodes then creating elements
+# bewtween nodes.
+
+# Define nodes
+mapdl.n(1)
+mapdl.n(5, 480)
+mapdl.n(6, 60, "1 $ N", 10, 420, 1)
+mapdl.fill(1, 5)
+mapdl.fill(6, 10)
+print(mapdl.nlist())  # List of the created nodes.
+
+# Define elements
+
+mapdl.e(1, 2, 6)
+mapdl.egen(4, 1, 1)
+print(mapdl.elist())  # List of the created elements.
+
+# Display elements with their nodes numbers.
+
+mapdl.eplot(show_node_numbering=True, line_width=5, cpos="xy") 
+
+###############################################################################
+# Define Boundary Conditions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Application of boundary conditions.
+
+mapdl.d(2, "UX", lab2="UY") 
+mapdl.d(4, "UY")
+mapdl.nsel("S", "LOC", "Y", 0)
+mapdl.d("ALL", "UZ")
+mapdl.d("ALL", "ROTX")
+mapdl.d("ALL", "ROTY")
+mapdl.nsel("ALL")
+
+###############################################################################
+# Define Distributed Loads
+# ~~~~~~~~~~~~~~~~~~~~~~~~~
+# Application a distributed force of :math:`w = (10000/12) lb/in` 
+# in the y-direction.
+
+w = 10000/12  # (lb/in) - distributed load.
+mapdl.sfbeam(1, 1, "PRES", w)
+mapdl.sfbeam(4, 1, "PRES", w)
+mapdl.finish()
+
+###############################################################################
+# Solve
+# ~~~~~
+# Enter solution mode and solve the system.
+
+mapdl.run("/SOLU")
+out = mapdl.solve()
+mapdl.finish()
+print(out)
+
+result = mapdl.result
+print(result)
+
+
+###############################################################################
+# Post-processing
+# ~~~~~~~~~~~~~~~
+# Enter post-processing. To get the stress and deflection results
+# from the middle node and cross-section of the beam we can use
+# mapdl.get().
+
+mapdl.post1()  # Start Post-Processing
+mapdl.set(1)  # Select first Load Step
+
+# Define Maximum Stress
+
+s_eqv_max_str = (mapdl.get('stress_eqv', 'secr', 2, 's', 'eqv', 'max')).split()
+s_eqv_max = -float(s_eqv_max_str[0])
```

Comment:
We should be using `get_value` here. If this is broken, we need to open a new issue and create a PR to fix this.